integration/redis_test.go

```diff
@@ -52,7 +78,171 @@ func (s *RedisSuite) setupStore(c *check.C) {
 func (s *RedisSuite) TestSimpleConfiguration(c *check.C) {
 	s.setupStore(c)
 
-	file := s.adaptFile(c, "fixtures/redis/simple.toml", struct{ RedisAddress string }{s.redisAddr})
+	endpoints := strings.Join(s.redisEndpoints, ",")
+	file := s.adaptFile(c, "fixtures/redis/simple.toml", struct{ RedisAddress string }{endpoints})
```

Comment:
```suggestion
```suggestion
	file := s.adaptFile(c, "fixtures/redis/simple.toml", struct{ RedisAddress string }{
		RedisAddress: strings.Join(s.redisEndpoints, ","),
	})
```
```